src/error.rs

```diff
@@ -1,14 +1,54 @@
 use http::{Method, Uri};
 use hyper::Error as HyperError;
-use snafu::Snafu;
+use std::{error::Error, fmt};
```

Comment:
please import the actual names and not a module
```suggestion
use std::{error::Error, fmt::{Display, Formatter, Result as FmtResult}};
```